app/src/main/java/com/example/cm/ui/InviteFriends/InviteFriendsFragment.java

```diff
@@ -136,4 +135,4 @@ public void onCheckBoxClicked(String id) {
     public void onItemClicked(String id) {
         // do nothing
```

Comment:
Shouldn't this onItemClicked handler to the same as in the FriendsFragment? Since one could also want to view profiles in that list